strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/cron/jobs/RegenerateMavenChecksumCronJobTestIT.java

```diff
@@ -145,22 +145,22 @@ public void testRegenerateArtifactChecksum()
         getLayoutProvider(REPOSITORY_SNAPSHOTS).rebuildMetadata(STORAGE0, REPOSITORY_SNAPSHOTS,
                                                                 "org/carlspring/strongbox/strongbox-checksum-one");
 
-        FileUtils.deleteIfExists(new File(snapshotArtifact_1.getFile()
-                                                            .getAbsolutePath() + ".md5"));
-        FileUtils.deleteIfExists(new File(snapshotArtifact_1.getFile()
-                                                            .getAbsolutePath() + ".sha1"));
-        FileUtils.deleteIfExists(new File(snapshotArtifact_1.getFile()
-                                                            .getAbsolutePath()
-                                                            .replaceAll("jar", "pom") + ".md5"));
-        FileUtils.deleteIfExists(new File(snapshotArtifact_1.getFile()
-                                                            .getAbsolutePath()
-                                                            .replaceAll("jar", "pom") + ".sha1"));
+        TestFileUtils.deleteIfExists(new File(snapshotArtifact_1.getFile()
```

Comment:
This looks good, but I think it would be even better to just do a `static import` for the necessary method. This way there'll be less unnecessary code to go through.